go.mod

```diff
@@ -15,6 +14,7 @@ require (
 	github.com/moby/sys/user v0.3.0
 	github.com/moby/sys/userns v0.1.0
 	github.com/mrunalp/fileutils v0.5.1
+	github.com/opencontainers/cgroups v0.0.0-00010101000000-000000000000
```

Comment:
And update this too, probably

Reply:
Done. We have not yet made a release in there, but we will soon. For now, use an unversioned dependency.

Reply:
I created a v0.0.1 tag, so this is now resolved.